lib/include/tick/array/vector/cblas.h

```diff
@@ -1,6 +1,24 @@
 
```

Comment:
I like the way you've called mkl ops_mkl, why not ops_blas and ops_unoptimized ?